image/image.go

```diff
@@ -261,13 +354,11 @@ func bootstrapToRootDir(sto Store, model *asserts.Model, opts *Options) error {
 
 		// set seed.yaml
 		seedYaml.Snaps = append(seedYaml.Snaps, &snap.SeedSnap{
-			Name:        info.Name(),
-			SnapID:      info.SnapID,
-			Revision:    info.Revision,
-			Channel:     info.Channel,
-			DeveloperID: info.DeveloperID,
-			Developer:   info.Developer,
-			File:        filepath.Base(fn),
+			Name:       info.Name(),
+			SnapID:     info.SnapID,
+			Channel:    info.Channel,
+			File:       filepath.Base(fn),
+			Sideloaded: info.SnapID == "",
```

Comment:
I've been trying to kill this term for months, but it seems to show up again nevertheless. :)

What do we mean by sideloaded here?  Let's please translate it into a more meaningful term to us and to other people looking at it.